File: data/multilang_comparison/program_cpp_vector/src/func47.cpp
#include "program_cpp_vector.hpp" 
Array* func47(Array_param* vars, int loopsFactor) {
   size_t pCounter = vars->size;
   Array* array11;
   if (pCounter > 0) {
      array11 = vars->data.at(--pCounter);
      array11->refC++;
      DEBUG_COPY(array11->id);
   } else {
      array11 = new Array();
      array11->size = 919;
      array11->refC = 1;
      array11->id = 11;
      array11->data = std::vector<unsigned int>(array11->size, 0);
      DEBUG_NEW(array11->id);
   }
   return array11;
}

File: data/multilang_comparison/program_julia/src/func47.jl
include("program_julia_head.jl")
function func47(vars::JArray_param, loopsFactor::Int)::JArray
   pCounter = length(vars.data)
   array11 = nothing
   if pCounter > 1
       array11 = vars.data[pCounter]
       pCounter -= 1
       array11.refC += 1
   else
       array11 = JArray(zeros(Int, 919), 919, 1, 11)
   end
   return array11;
end

